packages/@icon-magic/config-reader/src/schemas/config-schema.ts

```diff
@@ -3,14 +3,61 @@ const variantProperties = {
   path: { type: 'string', minLength: 1 }
 };
 
+const pluginProperties = {
+  name: { type: 'string' },
+  iterants: {
+    type: ['array'],
+    items: {
+      type: 'string'
+    }
+  },
+  params: ['object'],
+  writeToOutput: {
+    type: 'boolean',
+    default: true
```

Comment:
I think we default to false here.

Reply:
ah ok, then the interface needs to be updated: `writeToOutput?: boolean; // Do not set unless for debugging the plugin. By default, this is set to true for the last plugin of each step - build, generate, etc` https://github.com/linkedinlabs/icon-magic/blob/master/packages/%40icon-magic/icon-models/src/interface.ts#L39

Reply:
will wait on your confirmation and update.

Reply:
It's still optional from the config's perspective. I've taken care of the defaults in the code. It's false by default for all plugins (unless specified in the config), but true for the last plugin in each phase.

Reply:
updated!